src/services/services.ts

```diff
@@ -2912,6 +2919,63 @@ module ts {
 
                 return filteredMembers;
             }
+
+            function isExternalModuleName(token: Node): boolean {
+                if (token && token.parent && token.kind === SyntaxKind.StringLiteral && isInStringOrRegularExpressionOrTemplateLiteral(token)) {

```

Comment:
`isInStringOrRegularExpressionOrTemplateLiteral` is a local helper function that captures the position of the cursor and determines if the cursor is inside of any of the implied literals.